docs/rackspace-infrastructure-deployment.md

```diff
@@ -0,0 +1,83 @@
+
+​	For example, a customer in their own datacenter can stand up the reference architecture hardware for Rackspace Flex and, granting ingress access to DRAC or iLO, allow Rackspace engineers to create the Ironic environment and flavors required for the compute hardware.  The servers are identified their first PXE boot and the proper server flavor and image are applied.  When a server goes offline to due hardware or drive failures, it can be re-provisioned after repair back to its original operationg system and added back to the Openstack environment through Flex automation.
+
+### Leaf-Spline Network Architecture
+
+#### Overview
+
+​	Spine-leaf, or leaf-spine, is a two-layer network topology composed of spine and leaf switches. A spine-leaf architecture helps data center networks reduce network latency and hop count and improve network efficiency.  This two-layer full-mesh network topology was designed as an alternative to three-tier architectures and is suitable for modern data centers  that experience more east-west network traffic than north-south or switch-to-switch traffic. East-west traffic flows transfer data packets from server to server within a data center.
+
+- **Spine switches.** Leaf switches connect to spine switches and mesh into the spine, forming the access layer that delivers network connection points for servers.
+- **Leaf switches.** Servers and storage connect to leaf  switches and consist of access switches that aggregate traffic from  servers. They connect directly to the spine.
+
+#### Advantages of Leaf-Spline Architecture
+
+- **Redundancy.** Each leaf switch connects to every spine  switch, which increases the amount of redundancy while also reducing  potential bottlenecks.
+- **Lower latency.** Spine-leaf minimizes latency and bottlenecks because each payload only has to travel to a spine switch and to another leaf switch (two hops) to reach its endpoint.
+- **Performance.** Protocols such as Shortest Path Bridging  (SPB) and Transparent Interconnection of Lots of Links (TRILL) aid in  avoiding traffic congestion.
+- **Scalability.** Additional spine switches can be added to help avoid oversubscriptionand increase scalability.
+- **Reduces spending.** A spine-leaf architecture increases  the number of connections each switch can handle, so data centers  require fewer devices to deploy.
+
+![image-20241018150739704](./assets/images/spine-leaf.png.png)
+
+
+
+#### Rackspace Network Design Details 
+
+​	Rackspace utilizes Spline-leaf network architecture where server to server traffic (east-west) has higher importance than external connectivity of the deployed application.  This is ideal for single or multi tenant deployments that process large workloads of internal data kept in **block** or **object** storage.
```

Comment:
Please make the document more agnostic outside Rackspace and we should start with a multi tenant cloud should implement a leaf-spine model etc